### PR TITLE
fix(ci): Run CI on chrysalis-pt-2 branch

### PR DESCRIPTION
**Pull request by rajivshah3**
# Description of change

When the default branch was changed to `chrysalis-pt-2`, the CI configs were not updated. As a result, they don't run on PRs or pushes to the `chrysalis-pt-2` branch

## Links to any relevant issues

N/A

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

N/A

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by rajivshah3** [comment]
`rustfmt` errors are fixed in https://github.com/iotaledger/bee/pull/396

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -27,6 +29,13 @@ jobs:
         toolchain: ${{ matrix.rust }}
         override: true
 
+    - name: Set up MSYS2 and install clang
+      if: matrix.os == 'windows-latest'
+      uses: msys2/setup-msys2@v2
+      with:
+        update: true
+        install: mingw-w64-x86_64-clang
+
```
Comment (Alex6323) [inline .github/workflows/test.yml:38]: is this also related to this issue?

Comment (rajivshah3) [reply .github/workflows/test.yml:38]: Not really, but the pipeline is now failing because it can't compile RocksDB

**Comment by thibault-martinez** [comment]
To be honest, CI was never put on cpt2 on purpose. This branch is temporary to allow us to be quick and not always have to care about some aspects of the CI. CI will be taken care of when merging to dev. Also, this branch is disappearing very soon...

**Comment by thibault-martinez** [comment]
We are decommissioning cpt2 at the end of the week so this won't be necessary anymore, thanks anyway!
